sspl_test/alerts/realstor/test_real_stor_for_platform_sensor.py

```diff
@@ -108,22 +83,13 @@ def test_real_stor_sensor_voltage(agrs):
 
 
 def test_real_stor_sensor_temperature(agrs):
-    check_sspl_ll_is_running()
-    enclosure_sensor_message_request("ENCL:enclosure:sensor:temperature", "*")
-    enclosure_sensor_msg = None
-    time.sleep(4)
-    while not world.sspl_modules[IngressProcessorTests.name()]._is_my_msgQ_empty():
-        ingressMsg = world.sspl_modules[IngressProcessorTests.name()]._read_my_msgQ()
-        time.sleep(2)
-        print("Received: %s" % ingressMsg)
-        try:
-            msg_type = ingressMsg.get("actuator_response_type")
-            if msg_type["info"]["resource_type"] == "enclosure:sensor:temperature":
-                temperature_module_actuator_msg = msg_type
-                break
-        except Exception as exception:
-            time.sleep(4)
-            print(exception)
+    instance_id = "*"
+    resource_type = "storage:hw:platform_sensor:temperature"
```

Comment:
conflicting resource_id at line `105` realstor_encl.py,  https://github.com/Seagate/cortx-monitor/pull/502/files#diff-51ae3526f81a647428340696c8e48fd6b68a095316b7c1331da778cc191b4825R105

Although we are checking 2nd last entry after splitting by `:`, but resource_id has the `ENCL` prefix there, and it is not present here..

Reply:
Let me know the conflict. Prefix ENCL is added in next line.
`send_enclosure_actuator_request("ENCL:%s" % resource_type, instance_id)`

With index -2, we identify the component type `platform_sensor`.

Reply:
Yes sorry, so `ENCL` is there as a prefix for requests. Got it 👍